folly/portability/Time.cpp

```diff
@@ -274,6 +274,8 @@ extern "C" int clock_gettime(clockid_t clock_id, struct timespec* tp) {
       return -1;
   }
 }
+#elif defined(__linux__)
+#include <time.h>
```

Comment:
What is this for? The include is not used below.

Reply:
Moreover, it's already included in the `<Time.h>` header in `folly/portability`.